src/main/java/com/google/sps/classes/SqlConstants.java

```diff
@@ -18,26 +18,6 @@
  * SQL constants used throughout different queries.
  */
 public final class SqlConstants {
-  // Constants used for querying a question.
-  public static final int QUESTION_FETCH_ID = 1;
-  public static final int QUESTION_FETCH_TITLE = 2;
-  public static final int QUESTION_FETCH_BODY = 3;
-  public static final int QUESTION_FETCH_ASKERID = 4;
-  public static final int QUESTION_FETCH_ASKERNAME = 8;
-  public static final int QUESTION_FETCH_DATETIME = 5;
-  public static final int QUESTION_FETCH_NUMBEROFFOLLOWERS = 7;
-  public static final int QUESTION_FETCH_NUMBEROFANSWERS = 11;
-
-  // Constants used for inserting a question.
-  public static final int QUESTION_INSERT_TITLE = 1;
-  public static final int QUESTION_INSERT_BODY = 2;
-  public static final int QUESTION_INSERT_ASKERID = 3;
-
-  // Constants used for inserting a follower.
-  public static final int QUESTION_FETCH_MAXID = 1;
-  public static final int FOLLOWER_INSERT_QUESTIONID = 1;
-  public static final int FOLLOWER_INSERT_ASKERID = 2;
-
```

Comment:
Can you please update this file to avoid any unrelated changes? A lot of these changes are just identical deletions/insertions. I'm guessing this is just from merging and branching, but it would be great to take a look at the "Files changed" before submitting for review so that it makes your reviewers' lives easier :)

Reply:
I tried to reorder, but I think that is best left for another PR. Undid those changes!